app/src/main/res/layout/sheet_custom_drug_entry.xml

```diff
@@ -0,0 +1,116 @@
+    android:minHeight="@dimen/spacing_0"
+    android:text="@string/custom_drug_entry_sheet_remove"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:visibility="visible" />
+
+  <LinearLayout
+    android:id="@+id/textFieldsLinearLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginHorizontal="@dimen/spacing_16"
+    android:layout_marginStart="@dimen/spacing_36"
+    android:layout_marginTop="@dimen/spacing_16"
+    android:layout_marginEnd="@dimen/spacing_36"
+    android:layout_marginBottom="@dimen/spacing_16"
+    android:baselineAligned="true"
+    android:orientation="horizontal"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@+id/titleTextView">
+
+    <com.google.android.material.textfield.TextInputLayout
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_weight="1"
+      app:hintTextAppearance="?attr/textAppearanceBody2">
+
+      <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/drugDosageEditText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/custom_drug_entry_sheet_dosage"
+        android:imeOptions="actionNext"
+        android:inputType="textCapWords"
+        android:textColor="@color/color_on_surface_67" />
+
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <com.google.android.material.textfield.TextInputLayout
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="@dimen/spacing_8"
+      android:layout_weight="1"
+      app:hintTextAppearance="?attr/textAppearanceBody2"
+      app:hintTextColor="?attr/colorPrimary"
+      app:hintEnabled="true"
+      android:textColorHint="?attr/colorPrimary">
+
+      <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/drugFrequencyEditText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/custom_drug_entry_sheet_frequency"
+        android:imeOptions="actionDone"
+        android:textColor="@color/color_on_surface_67" />
+
+    </com.google.android.material.textfield.TextInputLayout>
+
+  </LinearLayout>
+
+  <FrameLayout
+    style="@style/Widget.Simple.Button.Frame"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/spacing_32"
+    app:layout_constraintTop_toBottomOf="@+id/textFieldsLinearLayout">
+
+    <com.google.android.material.button.MaterialButton
+      android:id="@+id/saveButton"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:text="@string/customprescription_save_drug"
+      app:cornerRadius="@dimen/spacing_0"
```

Comment:
I don't think we need to remove the corner radius, based on the designs it has a 2dp corner radius and default button style will take care of it. Can we remove this, please?